src/sass/components/_product-box.scss

```diff
@@ -78,5 +80,36 @@
     display: flex;
     justify-content: space-between;
     align-items: center;
+
+    .price {
+      display: flex;
+      justify-content: flex-end;
+
+      .old-price {
+        display: flex;
+        justify-content: flex-end;
+        font-size: 12px;
+        color: #a5a5a5;
+        font-weight: 300;
+        text-align: center;
+        line-height: 24px;
+        padding: 5px 10px;
+        position: relative;
+
+        &::after {
+          content: "";
+          display: inline-block;
+          width: 39px;
+          height: 1px;
+          position: absolute;
+          top: 0;
+          bottom: 0;
+          left: 0;
+          right: 0;
+          margin: auto;
+          background-color: #292929;
+        }
+      }
+    }
   }
-}
+}
```

Comment:
Niepotrzebnie usuwasz pustą linie na końcu pliku

Reply:
Zmieniłam to w kolejnym commicie. Poprawiłam też na przyszłość w konfiguracji edytora.